Team6220_2021/src/main/java/org/firstinspires/ftc/team6220_2021/TickTest.java

```diff
@@ -0,0 +1,55 @@
+package org.firstinspires.ftc.team6220_2021;
+
+import com.qualcomm.robotcore.eventloop.opmode.Autonomous;
+import com.qualcomm.robotcore.eventloop.opmode.Disabled;
+import com.qualcomm.robotcore.eventloop.opmode.TeleOp;
+import com.qualcomm.robotcore.hardware.DcMotor;
+import com.qualcomm.robotcore.hardware.DcMotorSimple;
+import com.qualcomm.robotcore.hardware.Servo;
+@Disabled
+@TeleOp(name = "TickTest", group = "TeleOp")
+public class TickTest extends MasterOpMode{
+    DcMotor motorBackLeft;
+    DcMotor motorBackRight;
+    DcMotor motorFrontLeft;
+    DcMotor motorFrontRight;
+    DcMotor motorDuck;
+    DcMotor motorArm;
+    Servo servoGrabber;
+    Servo servoArm;
+    @Override
+    public void runOpMode() {
+        motorBackLeft = hardwareMap.dcMotor.get("motorBackLeft");
```

Comment:
You should not define your motors here.  They should already be defined in MasterOpMode.  Don't define them more than once.  Don't reverse motors here.  Reverse them once, and stick to the same convention throughout all programs.